<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FishGame</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelturn">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>20</y>
      <width>420</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Aharoni&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelback">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>440</width>
      <height>640</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelid">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>60</y>
      <width>420</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Aharoni&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labellv">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>100</y>
      <width>420</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Aharoni&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelhp">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>150</y>
      <width>420</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Aharoni&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelatt">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>180</y>
      <width>420</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Aharoni&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelsp">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>220</y>
      <width>420</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Aharoni&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labeltest">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>260</y>
      <width>420</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Aharoni&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelout1">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>600</y>
      <width>420</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 8pt &quot;Aharoni&quot;;
text-decoration: underline;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelout3">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>540</y>
      <width>420</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Simplified Arabic&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelout5">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>480</y>
      <width>420</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Simplified Arabic&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelout4">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>510</y>
      <width>420</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Simplified Arabic&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelout2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>570</y>
      <width>420</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Simplified Arabic&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonstop">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>310</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Aharoni&quot;;
</string>
    </property>
    <property name="text">
     <string>stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>410</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Aharoni&quot;;</string>
    </property>
    <property name="text">
     <string>result</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonrelease">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>360</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Aharoni&quot;;</string>
    </property>
    <property name="text">
     <string>return</string>
    </property>
   </widget>
   <zorder>labelback</zorder>
   <zorder>labelturn</zorder>
   <zorder>labelid</zorder>
   <zorder>labellv</zorder>
   <zorder>labelhp</zorder>
   <zorder>labelatt</zorder>
   <zorder>labelsp</zorder>
   <zorder>labeltest</zorder>
   <zorder>labelout1</zorder>
   <zorder>labelout3</zorder>
   <zorder>labelout5</zorder>
   <zorder>labelout4</zorder>
   <zorder>labelout2</zorder>
   <zorder>buttonstop</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>buttonrelease</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
